tasks/main.yml

```diff
@@ -65,6 +65,13 @@
 
 - import_tasks: download.yml
 
+- name: Add Python Magic to requirements
+  become: yes
+  become_user: "{{ odoo_role_odoo_user }}"
+  lineinfile:
```

Comment:
Why don't we install the library directly with the `pip` module?

Reply:
It's another option. I didnt't realize that this module could fit there.